apps/meteor/app/theme/client/imports/general/base.css

```diff
@@ -54,7 +48,7 @@ body {
 		display: table;
 		clear: both;
 
-		content: "";
+		content: '';
```

Comment:
`question` There is no stylelint rule for this?

Reply:
to be honest I would be very happy deleting/removing this whole file, and every time I save it changes the value, that time I just gave up